VK_App/VK_App/Session/NetworkService.swift

```diff
@@ -22,49 +19,50 @@ struct NetworkService {
         static let searchQueryText = "&q="
         static let versionText = "&v=5.81"
     }
-    
+
     // MARK: - Public Methods
+
     func getInfo(path: String) {
         let url = "\(Constants.baseURL)\(path)"
         AF.request(url).responseJSON { response in
             guard let value = response.value else { return }
             print(value)
         }
     }
-    
+
     func getFriends() {
```

Comment:
```suggestion
    func fetchFriends() {
```
у нас могут быть get post put запросы, лучше для запроса писать fetch

Reply:
и ниже также